content/english/supply-chain-analytics/SolverAndSolving.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Solving Your Problem"
+date: 2021-03-13T14:17:07.42-07:00
+draft: false
+weight: 4
+---
+
+To run an optimization problem in Excel, you need to enable the add-on called “Solver.” Follow the link below for details or see the image below. 
+
+![Screenshot of steps to download Excel solver](img/Download%20Excel%20solver%20img.jpg)
+
+Click [here](
+https://support.microsoft.com/en-us/office/load-the-solver-add-in-in-excel-612926fc-d53b-46b4-872c-e24772f078ca) for more help if you are having trouble.
+
+You can find your solver add-in by clicking on “data” and then “solver” in the header. 
```

Comment:
I wonder if it makes more sense to add more explicit instructions on how to follow along the Excel file, including screenshots of the sheet and places where students should be entering data and/or formulas